### PR TITLE
Jetpack Cloud: Update Manifest.json

### PR DESCRIPTION
**Pull request by cbauerman**
#### Changes proposed in this Pull Request

* Use Jetpack.com colors/names in Manifest.json

Screen Shot 2020-05-20 at 5 44 28 PM

#### Testing instructions

1. Boot Jetpack Cloud and click the Install button in Chrome

Screen Shot 2020-05-20 at 5 44 11 PM

2. Verify the values are for "Jetpack.com" and not "WordPress.com"
3. Boot Calypso and verify the install is for WordPress.com
Screen Shot 2020-05-20 at 5 46 21 PM

### CODE REVIEW

File: client/server/pwa/manifest.js [hunk 1]
```diff
@@ -2,10 +2,62 @@
  * External dependencies
  */
 import querystring from 'querystring';
+import config from 'config';
```
Comment (ChaosExAnima) [inline client/server/pwa/manifest.js:5]: This is an internal dependency, I believe?

Comment (cbauerman) [reply client/server/pwa/manifest.js:5]: yes it is 😛

**Comment by mattgawarecki** [comment]
Is the color of the app's title bar supposed to match the product? Here's what I see for the Jetpack Cloud app:

image

And here's the one for WordPress.com:

image

**Comment by cbauerman** [comment]
> Is the color of the app's title bar supposed to match the product?

The `theme_color` should take care of that 🤔 let me take a look...

**Comment by cbauerman** [comment]
@mattgawarecki when you get a change you should give it another try. There was a `<head>` element that overriding the manifest

**Comment by cleacos** [comment]
The colors match with the Jetpack design. If I try to authorize the app, I got this error:

image

I suppose that it's not part of this PR (just in case)

**Pull request merged**